README.md

```diff
@@ -1,56 +1,45 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Kanban board: Car collector
 
+> Customers who wish to rent a car can use CarCollectors' website. Using the Vehicles API, the visitor may explore a vast selection of stunning cars. It also allows users to like, comment on, and reserve a certain automobile. The Involvement API is used to save all data.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- html
+- css
+- javascript
+- jest
+- webpack
 
```

Comment:
- [ ] You should record a video presenting your project and its features. Please kindly do that.